spec/jobs/reports/monthly_key_metrics_report_spec.rb

```diff
@@ -11,14 +11,16 @@
   let(:report_folder) do
     'int/monthly-key-metrics-report/2021/2021-03-02.monthly-key-metrics-report'
   end
-  let(:account_reuse_s3_path) do
-    "#{report_folder}/account_reuse.csv"
-  end
-  let(:total_profiles_s3_path) do
-    "#{report_folder}/total_profiles.csv"
-  end
-  let(:account_deletion_rate_s3_path) do
-    "#{report_folder}/account_deletion_rate.csv"
+  let(:account_reuse_s3_path) { "#{report_folder}/account_reuse.csv" }
+  let(:total_profiles_s3_path) { "#{report_folder}/total_profiles.csv" }
+  let(:account_deletion_rate_s3_path) { "#{report_folder}/account_deletion_rate.csv" }
+  let(:total_user_count_s3_path) { "#{report_folder}/total_user_count.csv" }
```

Comment:
I realized this didn't actually need to be `do...end` blocks for length so I compacted them. Tell me if you hate this.

Reply:
compact is good!